apps/src/lab2/views/dialogs/StartOverDialog.tsx

```diff
@@ -6,28 +6,37 @@ import {useAppSelector} from '@cdo/apps/util/reduxHooks';
 import {TEXT_BASED_LABS} from '@cdo/apps/lab2/types';
 const commonI18n = require('@cdo/locale');
 
+// Lab-specific messages for starting over.
+const LAB_SPECIFIC_MESSAGES: {[key: string]: string} = {
+  aichat: commonI18n.startOverAichatModelCustomizations(),
+};
+
 /**
  * Start Over dialog used in Lab2 labs.
  */
 const StartOverDialog: React.FunctionComponent<BaseDialogProps> = ({
   handleConfirm,
   handleCancel,
 }) => {
-  const currentAppName = useAppSelector(
-    state => state.lab.levelProperties?.appName
-  );
-  const isTextWorkspace =
+  const currentAppName =
+    useAppSelector(state => state.lab.levelProperties?.appName) || '';
+
+  const textBasedLab =
```

Comment:
The original `isTextWorkspace` is probably better, sorry I didn't see that original name when writing this snippet :)